app/assets/javascripts/email-analytics.js

```diff
@@ -0,0 +1,17 @@
+(function (global, GOVUK) {
+  'use strict'
+
+  window.GOVUK = window.GOVUK || {}
+
+  function canTrack () {
+    return !!GOVUK.analytics
+  }
+
+  // The EmailAnalytics module is a wrapper around GOVUK.analytics
+  GOVUK.EmailAnalytics = {
+    trackEvent: function trackEvent () {
+      if (!canTrack() || !GOVUK.analytics.trackEvent) { return }
```

Comment:
You could probably simplify this code a bit by removing the need for extra function and changing this to:

```suggestion
      if (!GOVUK.analytics || !GOVUK.analytics.trackEvent) { return }
```